replies.go

```diff
@@ -161,10 +161,10 @@ var replies = []Reply{
 	},
 	{
 		pattern: `(1\.16.*?(update|coming|support|release|impact|version|eta|when|out|support)|(update|coming|support|release|impact|version|eta|when|out|support).*?1\.16)`,
-		message: "No ETA on 1.16 Impact release, a message will be posted in <#" + announcements + "> when development starts & nightly builds.",
+		message: "No ETA on 1.16 Impact release, a message will be posted in <#" + announcements + "> when development starts & nightly builds. And please: https://cdn.discordapp.com/attachments/617140506069303306/741921114145685544/stfu30.gif",
 	},
```

Comment:
I wonder if using `![alt text](image URL)` syntax is better? At the very least a descriptive link is better than just pasting a massive URL into the message.

Perhaps the better solution is to add a way for bot replies to have an image attached, this wouldn't be particularly complex.